manim/mobject/svg/svg_mobject.py

```diff
@@ -11,6 +11,7 @@
 import warnings
 
 from xml.dom import minidom
+from colour import web2hex
```

Comment:
Good to know. Should this be made into an issue?

Reply:
yeah, can you make one?

Reply:
I just took a quick look at the module. While the code hasn't been updated in the past three years, there is still feedback on recent (Dec 2020) PRs. Perhaps it's because color parsing hasn't changed all that much?

Reply:
if that's the case, it's ok to depend on them.